github/commitstatus.go

```diff
@@ -198,11 +198,14 @@ func (cs CommitStatus) explainError(err error, state, sha, url string) error {
 // Example:
 // if hostname is: github.com it returns https://api.github.com
 // if hostname is overrriden in tests by localhost HTTP testserver it returns http://127.0.0.1:PORT
```

Comment:
// if hostname looks like a httptest server, it returns ...

Reply:
Fixed.